src/checkbox/checkbox.js

```diff
@@ -54,20 +54,30 @@ angular.module('angularify.semantic.checkbox', [])
                 scope.checked = true;
                 element.children()[0].setAttribute('checked', '');
             }
+            
+            //
+            // check if the parameter disabled is available
+            //
+            if (scope.disabled == 'disabled') {
```

Comment:
I think you forgot add `disabled` to the isolated scope. It should be declared in line 10-14 of this file.